app/client/components/svgs/tickIcon.tsx

```diff
@@ -1,10 +1,9 @@
 import React from "react";
 
-export const TickIcon = (props: any) => (
+export const TickIcon = () => (
```

Comment:
How come we removed this argument? Do we not have code that passes it anywhere anymore?